sw/device/lib/drivers/power_manager/power_manager.h

```diff
@@ -22,11 +22,44 @@ typedef enum power_manager_result {
   kPowerManagerError = 2,
 } power_manager_result_t;
 
+/**
+ * Domains.
+ */
+typedef enum power_manager_sel_domain {
+  kPeriph = 2,
+  kRam0   = 4,
+  kRam1   = 8,
+} power_manager_sel_domain_t;
+
+/**
+ * Domain states.
+ */
+typedef enum power_manager_sel_state {
+  kOn  = 0,
+  kOff = 1,
+} power_manager_sel_state_t;
+
 /**
  * Interrupt source.
  */
 typedef enum power_manager_sel_intr {
-  kTimer = 0,
+  kTimer_0 = 0,
+  kTimer_1 = 1,
+  kTimer_2 = 2,
+  kTimer_3 = 3,
+  kDma     = 4,
+  kGpio_0  = 5,
+  kGpio_1  = 6,
+  kGpio_2  = 7,
+  kGpio_3  = 8,
+  kGpio_4  = 9,
+  kGpio_5  = 10,
+  kGpio_6  = 11,
+  kGpio_7  = 12,
+  kExt_0   = 13,
```

Comment:
if you want to build a library sensitive to a parametrized number of external events, then this enum should be generated or parametrized

Reply:
Solved.